pravega-client-examples/src/main/java/io/pravega/example/secure/SecureReader.java

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright (c) 2019 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+package io.pravega.example.secure;
+
+import io.pravega.client.ClientConfig;
+import io.pravega.client.ClientFactory;
+import io.pravega.client.admin.ReaderGroupManager;
+import io.pravega.client.stream.*;
```

Comment:
Please change the import statement to use specific class.

Reply:
Fixed. Missed this. :) Thanks for spotting.